public-docs/swag-shop-03-custom-styling.md

```diff
@@ -0,0 +1,61 @@
+---
+id: swag-shop-3
+title: Custom Styling
+---
+
+# Adding custom styling
+
+## Styling a existing component
+All the CSS styles can be added directly in `lib/reactionTheme.js` file.
```

Comment:
[`lib/theme/reactionTheme.js`](https://github.com/reactioncommerce/reaction-next-starterkit/blob/v0.1.0/src/lib/theme/reactionTheme.js) file

*Note: I am linking to all files at the `v0.1.01` tag, not master - b/c that will break when we bump versions.

Reply:
Ignore this comment :D